src/org/voltdb/jsr107/CacheEventConsumer.java

```diff
@@ -110,7 +110,7 @@ public void run() {
 
         Properties props = new Properties();
         props.put("bootstrap.servers", kafkaHostnames);
-        props.put("group.id", cacheName + System.currentTimeMillis());
+        props.put("group.id", cacheName);
```

Comment:
We've discussed the cache naming here: https://github.com/srmadscience/voltdb-javacache-sandbox/pull/1/files#r1200610730

You've suggested to use the env variable. However, I've analyzed the code that actually potential user of cache library is already passing us the `cacheName` as a parameter, so I think we don't need to add env variable. This is consistent with what we've agreed to do here as well: https://github.com/srmadscience/voltdb-javacache/pull/1/files#r1200606124